src/zcl_minesweeper.abap

```diff
@@ -0,0 +1,75 @@
+class zcl_minesweeper definition
+  public
+  final
```

Comment:
try renaming this file, from "zcl_minesweeper.abap" to "zcl_minesweeper.clas.abap"